iguana/m_notary_KMD

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 # manually kill the iguana you need to restart first!
-# unlock any locked utxos. This will unlock utxos for both iguans, need a filter for address to only unlock the pubkey you need to restart. 
-komodo-cli lockunspent true `komodo-cli listlockunspent | jq -c .` 
+# unlock any locked utxos. This will unlock utxos for both iguans, need a filter for address to only unlock the pubkey you need to restart.
+komodo-cli lockunspent true `komodo-cli listlockunspent | jq -c .`
```

Comment:
This should use `komodo_binary` variable as well.

Reply:
No changes here. That’s an old script. My editor fixed white spaces there and commited it, sorry, my mistake

Reply:
It's not about whitespaces ... it's about `komodo-cli` binary. Script uses system-wide `komodo-cli` binary, but should use `komodo_binary` variable.

Reply:
That’s an old script not related to this PR. I will remove that from commit ASAP

Reply:
Please, keep this spaces fix in this PR. Editor insists on this fix :) Thanks.